packages/compass-query-bar/src/constants/query-properties.ts

```diff
@@ -6,9 +6,10 @@ const QUERY_PROPERTIES = [
   'sort',
   'skip',
   'limit',
-  'sample',
```

Comment:
I couldn't find a single place where `sample` was used, so I removed all associated logic completely. If someone knows where it's still used, please tell me